docs/getting-started-with-scalardb.md

```diff
@@ -46,16 +46,16 @@ Here is a database schema for the sample application. For the supported data typ
 }
 ```
 
```

Comment:
https://github.com/scalar-labs/scalardb/pull/94/files/725a1a0ad37bfee6e80c79118a5388f7f10eea6a..d44ac6477783459cd1b4976688e78e6f1e5a43f5#diff-72e7b9719f75bf45706d54341e54011b8e11cf8eef65dd2eff1b06ec580dba0dR3

Can you update line 3 with this?
```
Here we assume Oracle JDK 8 and Scalar DB storage implementation are properly configured.
If you haven't done it ...
```

Reply:
> Can you update line 3 with this?

yes

Thank you for your suggestion.
The document updated accordingly.